.circleci/config.yml

```diff
@@ -43,6 +43,8 @@ jobs:
             pip install -r requirements_optional.txt
             pip install torch
             pip install torchvision
+            pip install tensorflow
+            pip install keras
```

Comment:
Shouldn't be necessary as Keras is part of TF.

Reply:
I'm not sure.

Reply:
We don't need it for Travis or Jenkins. Try removing it.